ui/src/app/routing/app-routing.tsx

```diff
@@ -301,6 +307,14 @@ export const AppRoutingComponent: React.FunctionComponent<RoutingProps> =
             )}
           </React.Fragment>
         )}
+        {doesUserNeedToAcceptTOS && (
+          <AccountCreationTos
```

Comment:
I'm not really familiar with how app-router works, but: is this in the right place?

My assumption is that we desire to render this **instead of** any of the other components, but it appears to be possible to render this **in addition to** other components.  Is that right?

Reply:
My understanding (and i will confirm this) is that application while trying to match the url, finds doesUserNeedToAcceptTOS condition to match first so it stops right here and doesnt move forward